arelle/plugin/validate/ESEF/__init__.py

```diff
@@ -931,6 +981,17 @@ def checkMonetaryUnits(parent, relSet, visited):
             modelXbrl.warning("ESEF.RTS.Annex.II.Par.2.missingMandatoryMarkups",
                 _("Mandatory elements to be marked up are missing: %(qnames)s."),
                 modelObject=missingMandatoryElements, qnames=", ".join(sorted(str(qn) for qn in missingMandatoryElements)))
+            
+        # supplemental authority required tags
+        additionalTagQnames = set(qname(n, prefixedNamespaces)
+                                  for n in val.authParam.get("additionalMandatoryTags", ())
```

Comment:
Nit: Maybe slightly cleaner to filter on the `val.authParam` call but I don't feel strongly, would just be nice to avoid double creating the qname if possible